jujugui/static/gui/src/app/components/machine-view/machine-unit/_machine-unit.scss

```diff
@@ -0,0 +1,33 @@
+.machine-view__machine-unit {
+    position: relative;
+
+    &--uncommitted {
+        .machine-view__machine-unit-icon {
+            border: 1px solid $uncommitted;
+        }
+    }
+
+    &-icon {
+        display: inline-block;
+        margin-right: 10px;
+        padding: 2px;
+        border: 1px solid $canvas-background;
+        border-radius: 50%;
+
+        img {
```

Comment:
Could you select this element with a class. 

It's best to try and avoid selecting elements as they could change and could lose there styling. For example this could be replaced with an <svg> or <picture>-element in the future.